### PR TITLE
Frontend for package page

### LINKED ISSUE #2: [Frontend] Create package page
create a page to show the data created in #1

### PR DESCRIPTION
**Pull request by rajaFlutter**
Fixed #2

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,112 +1,92 @@
...
+                  </ul>
+                </div>
+              </div>
+              <div className="mt-5 grid grid-cols-2 gap-5 border-t border-gray-300 pt-5">
+                <button className="font-semibold text-blue-500 hover:underline text-start">
+                  Visit Source
+                </button>
+                <button className="font-semibold text-blue-500 hover:underline text-end">
+                  Visit Homepage
+                </button>
```
Comment (siddheshghadi751) [inline src/app/page.tsx:85]: Add homepage URL here

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,112 +1,92 @@
...
+                    {data.scores.health_risk.notes.map((note, noteIndex) => {
+                      return <li key={noteIndex}>{note}</li>;
+                    })}
+                  </ul>
+                </div>
+              </div>
+              <div className="mt-5 grid grid-cols-2 gap-5 border-t border-gray-300 pt-5">
+                <button className="font-semibold text-blue-500 hover:underline text-start">
+                  Visit Source
+                </button>
```
Comment (siddheshghadi751) [inline src/app/page.tsx:82]: add source URL here

**Review by siddheshghadi751 — CHANGES_REQUESTED** [review]

**Review by srossross — CHANGES_REQUESTED** [review]
can you please fix the linter warnings?

**Comment by siddheshghadi751** [comment]
> can you please fix the linter warnings?

linter warning is already fixed now

**Comment by srossross** [comment]
You have fixed the errors, but I'm still seeing the warnings:

image

https://medium.com/@dobinshin/how-to-use-eslint-plugin-tailwindcss-9b3aba12e5f5

Classnames Order:

    * Automatically sorts Tailwind CSS classes in a consistent order.
    * Enhances code readability.
    * Simplifies resolving merge conflicts.

**Review by siddheshghadi751 — APPROVED** [review]

**Pull request merged**